images/baremetal/Dockerfile.ci

```diff
@@ -6,7 +6,7 @@ ARG libvirt_version="8.0.0"
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.20-openshift-4.14 AS builder
 ARG libvirt_version
 ARG TAGS="libvirt baremetal"
-RUN dnf install -y libvirt-devel-$libvirt_version && \
+RUN dnf install --enablerepo=powertools -y libvirt-devel-$libvirt_version nmstate-devel && \
```

Comment:
What's the need to enable powertools? In agent-installer-utils we use just the base image [repos](https://github.com/openshift/agent-installer-utils/blob/main/images/Dockerfile.ocp)